CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,162 @@
+# Quansight Repository Code of Conduct
+
+1. Quansight Repository Code of Conduct Introduction
+  a. Expected Behavior
+  b. Unacceptable Behavior
+2. Incident Reporting Guidelines
+  a. How to Submit a Report
+  b. Conflicts of Interest
+  c. What to Include in a Report
+  d. Procedure after Filing a Report
+3. License  
+  
+------- 
+
+## 1. Quansight Repository Code of Conduct Introduction
+
+This Code of Conduct aims to empower all users, maintainers, and contributors of Quansight public repositories and open source projects to facilitate healthy and constructive community behavior. It outlines expectations for participation in Quansight-managed public repositories, as well as steps for reporting unacceptable behavior. 
+
+This code of conduct applies to all repos and communities for Quansight-managed open source projects regardless of whether or not the repository explicitly calls out its use of this code. The code also applies in public spaces when an individual is representing a project or its community. Examples include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
```

Comment:
Should perhaps clarify here that this also applies to Quansigh Labs (and also add this to the Quansight-Labs org once this is done).